Source/Modules/php.cxx

```diff
@@ -2007,6 +2013,10 @@ class PHP : public Language {
 	  if (value) {
 	    Printf(pragma_phpinfo, "%s\n", value);
 	  }
+	} else if (Strcmp(type, "version") == 0) {
+	  if (value) {
+	    Printf(pragma_version, "%s", value);
```

Comment:
This appends `value` to any existing contents of `pragma_version`, which is a bad way to handle multiple occurrences of this pragma.

I'd suggest initialising `pragma_version` to `NULL` by default, and then you can probably just assign `value` to it here (`value` points to a string in the parse tree which should stay valid while we need it).